res/layout/contact_selection_list_fragment.xml

```diff
@@ -21,12 +16,12 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:gravity="center|center_vertical"
-            android:layout_marginTop="15dp"
-            android:text="@string/one_moment"
+            android:padding="16dp"
+            android:lineSpacingMultiplier="1.3"
+            android:visibility="gone"
+            android:text="@string/contacts_empty_hint"
```

Comment:
Will this text ever be shown to the user? (`No contacts.`) I didn't find a place where I can see it.

Also, aren't we directly changing the text in onCreate? Nothing will be shown to the user until onCreate is done (?)

Reply:
no it is not shown currently, however, the layout is shared for different purposes, so having a more generic "empty" text makes some sense. before there was "one moment" which i found pretty confusing :) but maybe we should just set no text at all in the layout.

Reply:
i will move that to tools:text, i'll add a commit.